Sources/HipMobileUI/ViewModels/Views/ExhibitPreviewViewModel.cs

```diff
@@ -49,17 +49,17 @@ public ExhibitPreviewViewModel(Exhibit exhibit, INearbyExhibitManager exManager)
 
         public string ExhibitName { get; }
 
-        private void Accept()
+        async void Accept()
```

Comment:
Why is this and the `Deny` method not private anymore?

Reply:
Probably a typo. I will fix this.